tests/test_loadDiaCatalogs.py

```diff
@@ -84,7 +84,7 @@ def setUp(self):
 
         # These columns are not in the DPDD, yet do appear in DiaSource.yaml.
         # We don't need to check them against the default APDB schema.
-        self.ignoreColumns = ["band", "bboxSize", "isDipole"]
+        self.ignoreColumns = ["band", "bboxSize", "isDipole", "flags"]
```

Comment:
Why not just remove it from DiaSource.yaml?

Reply:
As above, because it would break the functionality of the bitpacking.